<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendDataToTcpUdpPortWidget</class>
 <widget class="QWidget" name="SendDataToTcpUdpPortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>145</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,5,1,2" columnminimumwidth="0,0,0,0">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="lblServer">
       <property name="text">
        <string>Server</string>
       </property>
       <property name="buddy">
        <cstring>cmbServer</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QComboBox" name="cmbServer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblProtocol">
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="cmbProtocol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>TCP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UDP</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblPort">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>514</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblInitPacket">
     <property name="text">
      <string>Init packet (string) to send</string>
     </property>
     <property name="buddy">
      <cstring>cmbInitPacket</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbInitPacket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnInitPacket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/loupe.png</normaloff>:/img/loupe.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblWaitForAnswer">
     <property name="text">
      <string>Wait for answer</string>
     </property>
     <property name="buddy">
      <cstring>spinWaitForAnswer</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="spinWaitForAnswer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSec">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>sec</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblFinalPacket">
     <property name="text">
      <string>Final packet (string) to send</string>
     </property>
     <property name="buddy">
      <cstring>cmbFinalPacket</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbFinalPacket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnFinalPacket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/loupe.png</normaloff>:/img/loupe.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
